systemtest/src/test/java/io/strimzi/systemtest/kafka/KafkaVersionsST.java

```diff
@@ -160,31 +159,25 @@ void testKafkaWithVersion(final TestKafkaVersion testKafkaVersion) {
 
         LOGGER.info("Sending and receiving messages via PLAIN -> SCRAM-SHA");
 
-        KafkaClients kafkaClients = new KafkaClientsBuilder()
-            .withTopicName(testStorage.getTopicName())
-            .withBootstrapAddress(KafkaResources.plainBootstrapAddress(testStorage.getClusterName()))
-            .withNamespaceName(testStorage.getNamespaceName())
-            .withProducerName(testStorage.getProducerName())
-            .withConsumerName(testStorage.getConsumerName())
-            .withMessageCount(testStorage.getMessageCount())
+        KafkaClients kafkaClients = ClientUtils.getInstantPlainClientBuilder(testStorage)
             .withUsername(kafkaUserWrite)
             .withConsumerGroup(readConsumerGroup)
             .build();
 
         resourceManager.createResourceWithWait(kafkaClients.producerScramShaPlainStrimzi());
-        ClientUtils.waitForProducerClientSuccess(testStorage);
+        ClientUtils.waitForInstantProducerClientSuccess(testStorage);
 
         kafkaClients = new KafkaClientsBuilder(kafkaClients)
                 .withUsername(kafkaUserRead)
                 .build();
 
         resourceManager.createResourceWithWait(kafkaClients.consumerScramShaPlainStrimzi());
-        ClientUtils.waitForConsumerClientSuccess(testStorage);
+        ClientUtils.waitForInstantConsumerClientSuccess(testStorage);
 
         LOGGER.info("Sending and receiving messages via TLS");
 
-        kafkaClients = new KafkaClientsBuilder(kafkaClients)
-            .withBootstrapAddress(KafkaResources.tlsBootstrapAddress(testStorage.getClusterName()))
+        kafkaClients = ClientUtils.getInstantTlsClientBuilder(testStorage)
+            .withConsumerGroup(readConsumerGroup)
```

Comment:
What is the benefit of creating new clients with this method, when you can pass there the builder and just replace the bootstrap address? At least FMPOV there is no need to change this.

Reply:
seemed to me more clean about what is going on but i guess it is basically the same so i will revert this.